alerter.c

```diff
@@ -11,22 +12,37 @@ int networkAlertStub(float celcius) {
     return 200;
 }
 
-void alertInCelcius(float farenheit) {
+int testNetworkAlertStub(float celcius) {

```

Comment:
Stub is part of production code in the same file. This will get compiled and get shipped to field along with production code. Hence it is necessary to keep test code and production code separate in order to exclude test code for deployment.

In case this is the production code then the method name needs to be different.